lambda/handlers/intentHandler.js

```
@@ -305,16 +311,29 @@ const PlayAdhanIntentHandler = {
     const prayerName = helperFunctions.getResolvedId(handlerInput.requestEnvelope, "prayerName");    
-    let audioToFetch = "converted-adhan.mp3";
-    if(prayerName === "0"){
-      audioToFetch = "converted-adhan-fajr.mp3";
-    }
-    let audioUrl = Alexa.escapeXmlCharacters(await getS3PreSignedUrl(audioToFetch));
+    const sessionAttributes =
+        handlerInput.attributesManager.getSessionAttributes();
+    const { persistentAttributes } = sessionAttributes;    
+    let audioName = "Adhaan";
+    let audioUrl = prayerName === "0"? adhaanRecitation[0].fajrUrl : adhaanRecitation[0].otherUrl;
+    if(persistentAttributes && persistentAttributes.favouriteAdhaan){
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)